### PR TITLE
Migrate perms.users.item.{post,put} MODPERMS-182

### PR DESCRIPTION
**Pull request by adamdickmeiss**
Also expand permissions to search for these in migration.
Make an adjustment to the Permission User's permissions
property to reflect reality: array of string.

### CODE REVIEW

File: src/main/java/org/folio/rest/impl/PermissionUtils.java [hunk 1]
```diff
@@ -33,8 +33,8 @@
   public static final String PERMS_USERS_ASSIGN_IMMUTABLE = "perms.users.assign.immutable";
   public static final String PERMS_USERS_ASSIGN_MUTABLE = "perms.users.assign.mutable";
   public static final String PERMS_USERS_ASSIGN_OKAPI = "perms.users.assign.okapi";
-  public static final String PERMS_PERMS_ALL = "perms.all";
   public static final String PERMS_OKAPI_ALL = "okapi.all";
+  protected  static final String [] PERMS_MIGRATE = { "perms.users.item.post", "perms.users.item.put" };
```
Comment (julianladisch) [inline src/main/java/org/folio/rest/impl/PermissionUtils.java:37]:
```suggestion
  protected static final String [] PERMS_MIGRATE = { "perms.users.item.post", "perms.users.item.put" };
```

**Review by julianladisch — APPROVED** [review]
Looks good, thanks!

**Comment by skoczko** [comment]
@adamdickmeiss let's hold off on releasing this until we get a chance to discuss. I am no longer convinced this should be merged.

**Comment by adamdickmeiss** [comment]
It is merged, but we'll revert :-)

**Pull request merged**